apps/storybook/package.json

```diff
@@ -32,8 +33,8 @@
   },
   "scripts": {
     "storybook": "start-storybook -p 6006 --no-open",
-    "build": "build-storybook",
-    "dev": "yarn storybook --quiet",
+    "build": "dotenv -v NODE_OPTIONS=--openssl-legacy-provider -- build-storybook",
```

Comment:
i dont think you need dotenv to set this var, might be wrong tho

Reply:
It's needed to do it in a cross platform way. Without this it will fail on windows.

Reply:
We would, but it might break some of our processes because of incorrectly declared deps.

Reply:
pnpm should have you covered there too! https://pnpm.io/package_json#pnpmpackageextensions

just an fyi